src/index.js

```diff
@@ -88,6 +108,7 @@ function macrosPlugin(
               babel,
               interopRequire,
               resolvePath,
+              options,
```

Comment:
@conartist6 @kentcdodds 
Shouldn't the `options` also be passed in another `applyMacros` invocation?

https://github.com/kentcdodds/babel-plugin-macros/blob/d11217e724ca00125ead59f82a564776d61b56bc/src/index.js#L146-L154

Reply:
Could you open a new issue and/or a pull request with a failing test? Thanks!

Reply:
@InvictusMB You are right. It would affect macros meant to be configured via plugin options which are imported with `const { sym } = require('path/to/macro')`. Nice catch.

Reply:
I have created #120 with the repro and fix for this issue.